Modules/Core/Transform/wrapping/itkResampleInPlaceImageFilter.wrap

```diff
@@ -0,0 +1,14 @@
+itk_wrap_class("itk::ResampleInPlaceImageFilter" POINTER)
+  itk_wrap_image_filter("${WRAP_ITK_SCALAR}" 2)
+  itk_wrap_image_filter("${WRAP_ITK_VECTOR_REAL}" 2)
+  itk_wrap_image_filter("${WRAP_ITK_COLOR}" 2)
+
+  # VectorImages
+  UNIQUE(to_types "UC;${WRAP_ITK_SCALAR}")
```

Comment:
Looking at https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/ImageGrid/wrapping/itkRegionOfInterestImageFilter.wrap, we probably don't need special treatment of UC.

Reply:
Thanks @dzenanz. Removed in 0bb11ae.